src/App.tsx

```diff
@@ -2,75 +2,87 @@ import React from 'react';
 import { GoodsList } from './components/GoodsList';
 import './App.scss';
 
-import { getAll, get5First, getRedGoods } from './api/goods';
+import {
+  getAll,
+  get5First,
+  getRedGoods,
+} from './api/goods';
```

Comment:
This file doesn't appear in the PR, add it

Reply:
This file is in PR. The link is working.